app/views_redesigned/async_property_stat_cards/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="card-deck d-block d-lg-flex card-lg-gutters-3 mb-3">
+  <%= render "/@shared/widgets/stat_card",
```

Comment:
you'll need to create this file and then change this throughout this file to 
```suggestion
  <%= render "/shared/widgets/stat_card",
```

Reply:
:heavy_check_mark: